cleo_plugins/MemoryOperations/MemoryOperations.cpp

```diff
@@ -542,7 +542,7 @@ class MemoryOperations
     static OpcodeResult __stdcall opcode_0AC7(CLEO::CRunningScript* thread)
     {
         auto resultType = thread->PeekDataType();
-        if (!IsVariable(resultType) && IsVarString(resultType))
+        if (!IsVariable(resultType) && !IsVarString(resultType))
```

Comment:
`IsVariable` should be called `IsVarNumber`

Reply:
case DT_VAR:
case DT_VAR_ARRAY:
case DT_LVAR:
case DT_LVAR_ARRAY:

Is it done this way as some arrays related stuff? Worth consideration as it can be what else if not an number.